XMonad/Actions/GroupNavigation.hs

```diff
@@ -16,17 +16,21 @@
 -- query.
 --
 -- Also provides a method for jumping back to the most recently used
--- window in any given group.
+-- window in any given group, and predefined groups.
 --
 ----------------------------------------------------------------------
 
 module XMonad.Actions.GroupNavigation ( -- * Usage
                                         -- $usage
+
+                                        -- * Utilities
```

Comment:
Should this 'Utilities' heading be moved down right above the export of `isOnAnyVisibleWS`?  Right now `$utilities` will generate some documentation that says "handy queries for use with `nextMatch` etc." but it seems strange to put that at the top of a section containing e.g. `nextMatch` itself.